### PR TITLE
Quick Start Guide: add text on handling the command 

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR adds the text on handling the command by the `Console` Process Manager.

### CODE REVIEW

File: docs/quick-start/index.md [hunk 1]
```diff
@@ -408,6 +406,87 @@ of the first field of the `Print` command?
...
+             start="*private void println(*" 
+             end="    }*"?>
+```java
+private void println(String userName, String text) {
+    String output = format("[%s] %s", userName, text);
+    System.out.println(output);
+}
+```
+
+Then, the command-handling method concludes producing the event message:
```
Comment (dmdashenkov) [inline docs/quick-start/index.md:471]:
```suggestion
Then, the command-handling method concludes by producing the event message:
```

File: docs/quick-start/index.md [hunk 2]
```diff
@@ -408,6 +406,87 @@ of the first field of the `Print` command?
...
+            .setUsername(username)
+            .setText(command.getText())
+            .vBuild();
+}
+```
+
+Let's review the method in details.
+
+The `@Assign` annotation tells that we assign this method to handle the command which the
+method accepts as the parameter. The method returns the event message `Printed`.
```
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply docs/quick-start/index.md:435]: I don't think we need to speak German here.

File: docs/quick-start/index.md [hunk 3]
```diff
@@ -408,6 +406,87 @@ of the first field of the `Print` command?
...
+
+The following code obtains the name of the user and the text to print from the received command,
+and then applies them to the state of the Process Manager. Instances of the `Console` class keep
+the text printed for each user.
```
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply docs/quick-start/index.md:439]: Here I refer to the state of the entity. It refers to why we call `builder()`.

File: docs/quick-start/index.md [hunk 4]
```diff
@@ -408,6 +406,87 @@ of the first field of the `Print` command?
...
+            .newBuilder()
+            .setUsername(username)
+            .setText(command.getText())
+            .vBuild();
+}
+```
+
+Let's review the method in details.
+
+The `@Assign` annotation tells that we assign this method to handle the command which the
```
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply docs/quick-start/index.md:434]: It's either `Command` or “command” in this context. It would be harder to understand when capitalized.

**Pull request merged**